sample-cap/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "./srv",
+    "outDir": "./dist/srv/srv",
+    "tsBuildInfoFile": "./dist/.tsbuildinfo",
+    "composite": true
+  },
+  "include": ["srv/**/*.ts"],
+  "exclude": ["dist/**/*", "**/*.test.ts", "node_modules/**/*"],
+  "references": [
+    { "path": "../packages/ai-api" },
+    { "path": "../packages/foundation-models" },
+    { "path": "../packages/orchestration" },
+    { "path": "../packages/langchain" }
```

Comment:
why referencing the libs from the local workspace? in the package.json those libs are already defined as dependencies. please remove

Reply:
Right, this was copy paste... Thanks and removed.